packages/locale/lang/my.ts

```diff
@@ -0,0 +1,184 @@
+export default {
+  name: 'my',
+  el: {
+    breadcrumb: {
+      label: 'Breadcrumb', // to be translated
```

Comment:
Is there a missing translation here?

Reply:
@btea  Apologies for the late reply! I’ve fixed that. Please feel free to make any additional adjustments as needed. Let me know if you need any further modifications!